src/index.js

```diff
@@ -4,14 +4,19 @@ import stop from './actions/stop';
 import restart from './actions/restart';
 import { version } from '../package.json';
 
+const isDarwin = process.platform === 'darwin';
+
 program
   .version(version)
   .description('A utility to help setup a docker development environment with host based routing');
 
 program
   .command('start')
+  .option('-p, --port <port>', 'port')
   .description('Pull and start the dotdocker containers and configure DNS')
-  .action(start);
+  .action(res => {
+    start(isDarwin ? res.port : 53);
```

Comment:
does this mean the user needs to provide a port if they are using MacOS? what is the default for `res.port`